data_transfer/generic_https_transfer.py

```diff
@@ -56,6 +54,8 @@ def main(
         j = batch.new_job(f"URL {idx} ({filename})")
         quoted_url = quote(url)
         j.command(GCLOUD_ACTIVATE_AUTH)
+        # catch errors during the CURL
+        j.command("set -euxo pipefail")
         j.command(
             f"curl -L {quoted_url} | gsutil cp - {os.path.join(output_path, filename)}"
```

Comment:
Don't you want to add some retry logic here?

Reply:
Not for the generic transfer, I'd rather it fail and you have to retry them manually - I don't want to accidentally cause a boat load of egress through lots of retries.